src/NUnitFramework/tests/Api/DefaultTestAssemblyBuilderTests.cs

```diff
@@ -0,0 +1,75 @@
+// ***********************************************************************
+// Copyright (c) 2018 Charlie Poole, Rob Prouse
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// ***********************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using NUnit.Framework.Internal;
+using NUnit.Tests;
+using NUnit.Tests.Assemblies;
+using NUnit.Tests.Singletons;
+
+namespace NUnit.Framework.Api
+{
+    public class DefaultTestAssemblyBuilderTests
+    {
+        private const string MOCK_ASSEMBLY_FILE = "mock-assembly.dll";
```

Comment:
IIRC all caps is in our coding standard. Has it changed? I definitely like caps at the head of a test class because it shows me what all the constant parameters of the test are at one glance.

Reply:
Just checked to confirm; the coding standard hasn't mentioned constants as far back as the current file goes. I believe we want to be more permissive of preferences now, too, so I'm happy with keeping it.